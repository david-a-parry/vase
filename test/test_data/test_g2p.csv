"gene symbol","gene mim","disease name","disease mim","DDD category","allelic requirement","mutation consequence",phenotypes,"organ specificity list",pmids,panel,"prev symbols","hgnc id","gene disease pair entry date"
SYMBOL_1,12345,"SICKNESS",633333,probable,biallelic,"loss of function",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
SYMBOL_2,678910,"FAKE CONDITION",622222,confirmed,monoallelic,"all missense/in frame",HP:0001636,Skin;Skeleton;Brain/Cognition,22426308,DD,SNF2L2,11098,"2015-07-22 16:14:13"
SYMBOL_3,54321,"FAKE SYNDROME",611111,confirmed,monoallelic,"dominant negative",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
SYMBOL_4,98765,"FAKE SYNDROME 2",600000,probable,biallelic,"uncertain",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
SYMBOL_5,12345,"SICKNESS",633333,probable,biallelic,"loss of function",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
SYMBOL_6,98765,"FAKE SYNDROME 3",600000,probable,biallelic,"all missense/in frame",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
SYMBOL_7,54321,"FAKE SYNDROME",611111,confirmed,monoallelic,"loss of function",HP:0000007;HP:0000482;HP:0000647;HP:0007906;HP:0000568;HP:0000589;HP:0000639;HP:0000518;HP:0001104,Eye;Ear,18423520,DD,,5017,"2015-07-22 16:14:07"
