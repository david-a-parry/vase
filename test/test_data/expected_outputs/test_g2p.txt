1:20-C/A
1:40-C/T
1:50-C/T
1:100-T/C
1:110-C/G
1:140-C/T
